server/routes/handlers/projects.js

```diff
@@ -128,32 +128,48 @@ module.exports = {
     (req, res) => {
       const paginated = !(req.query.format && req.query.format === 'csv');
       req.query.scopes = { event_id: req.params.eventId };
-      return projectController
-        .getExtended(req.query, paginated)
-        .then((projects) => {
-          const dateFormat = date => new Date(date).toLocaleDateString();
-          if (!paginated) {
-            res.setHeader('Content-Type', 'text/csv');
-            const data = projects.toJSON();
-            const fields = [
-              { label: 'Name', value: 'name' },
-              { label: 'Description', value: 'description' },
-              { label: 'Category', value: 'category' },
-              { label: 'Supervisor Email', value: 'supervisor.email' },
-              { label: 'Owner Email', value: 'owner.email' },
-              { label: 'Created At', value: row => dateFormat(row.createdAt) },
-              { label: 'Updated At', value: row => dateFormat(row.updatedAt) },
+      return projectController.getExtended(req.query, paginated).then((projects) => {
+        const dateFormat = date => new Date(date).toLocaleDateString();
+        if (!paginated) {
+          res.setHeader('Content-Type', 'text/csv');
+          const data = projects.toJSON();
+          let fields = [
```

Comment:
fields can stay a const : we don't reassign it. const doesn't stop mutation of objects or arrays, just re-assignement.

Reply:
Technically I am reassigning here, https://github.com/CoderDojo/coolest-platform/pull/57/files/486ef2ed7658acd88c15a466f7dee72bb5799a2d#diff-cce739c87129ebc32c0b1a10a5c326a9R160 when I concat the arrays together.

Reply:
Ah, true. Forgot concat doesn't mutate :|